iocage_lib/ioc_stop.py

```diff
@@ -338,7 +338,7 @@ def __stop_jail__(self):
             msg = f'  + Removing jail process FAILED:\n' \
                 f'{stop_err.decode("utf-8")}'
             iocage_lib.ioc_common.logit({
-                'level': 'EXCEPTION',
+                'level': 'ERROR',
```

Comment:
@skarekrow i don't like this change. A lot of things can go wrong here and we shouldn't be executing `poststop` in those cases. I do however like the case of manually handling the umount commands for the time being. We can add a note specifying why we have reverted to this and in future change it when the upstream has fixed it. Thoughts ?

Reply:
I can’t recall exactly, but I think we can do the unmounting before removal of the jail process, since all processes should be shut down. That may allow us to keep the exception and introduce the behavior.

Reply:
What we do in our iocage-port for now is:
```diff
--- iocage_lib/ioc_stop.py.orig	2019-05-23 17:54:09 UTC
+++ iocage_lib/ioc_stop.py
@@ -322,9 +322,6 @@ class IOCStop(object):
         # will be written on the next jail start/restart.
         jail_conf_file = Path(f"/var/run/jail.ioc-{self.uuid}.conf")
 
-        if jail_conf_file.is_file():
-            cmd.extend(['-f', str(jail_conf_file)])
-
         cmd.extend(['-r', f'ioc-{self.uuid}'])
 
         stop = su.Popen(
```

I know, neither this "pull request" nor this patch is a good solution, but it works for the moment.